src/pages/test-report/cat-adi-part2/components/vehicle-check/vehicle-check.html

```diff
@@ -26,12 +26,11 @@
       }"
     >
       <driving-faults-badge class="driving-faults" [count]="getDrivingFaultCount()"></driving-faults-badge>
-      <div class='competency-label'>Show me / Tell me</div>
-      <div class="fault-wrapper">
+      <div class='competency-label'>Show/Tell me</div>
```

Comment:
Cat D uses SM/TM when struggling for space, not sure if that should be considered here too

Reply:
agree that they should be consistant.